docs/source/system_reference_guide/custom-environments.ipynb

```diff
@@ -49,14 +49,19 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">This is too vague <code>mamba env update ARGUMENTS</code> 
</pre><p>users won't know what arguments. Reasonably that is actually conda/mamba docs but most people aren't going there (we should link to them).</p><p>So I think moving </p><h3>Updating an existing environment with a configuration file</h3><p>up to here actually does make sense.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/252/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Agreed. 

Instead of moving like you suggested, replaced it with 

```
mamba env update -f config.yml
```

and then pointed to the next sections for details on what that config file could look like.